src/components/ItemTable/ItemAvailability.tsx

```diff
@@ -13,6 +16,16 @@ interface ItemAvailabilityProps {
  * TODO: Add Feedback box, Due date, Available font styles
  */
 const ItemAvailability = ({ item }: ItemAvailabilityProps) => {
+  const feedback = useContext(FeedbackContext)
+
+  const openFeedbackBox = feedback?.onOpen
```

Comment:
is there a specific case where feedback would be returned undefined?

Reply:
only if the provider wasn't wrapping the component. can remove the optional chaining.